dimod/generators/satisfiability.py

```diff
@@ -164,24 +203,39 @@ def random_2in4sat(variables: typing.Union[int, typing.Sequence[dimod.typing.Var
     For valid solutions, two of the literals in each clause should ``+1`` and
     the other two should be ``-1``.
 
+    Each clause contributes -2 to the energy when the clause is satisfied,
+    and at least 0 when unsatisfied. The energy :math:`H(s)` for a spin 
+    assignment :math:`s` is thus lower bounded by :math:`E_{SAT}=-2`:code:`num_clauses`, 
+    this lower bound matches the ground state energy in satisfiable instances. 
+    The number of violated clauses is at most :math:`(H(s) - E_{SAT})/2`.
+
     .. [#] Adam Douglass, Andrew D. King & Jack Raymond,
        "Constructing SAT Filters with a Quantum Annealer",
        https://link.springer.com/chapter/10.1007/978-3-319-24318-4_9
 
     Args:
         num_variables: The number of variables in the problem.
         num_clauses: The number of clauses. Each clause contains three literals.
+        is_planted: Create literals uniformly subject to the constraint that the
+            all 1 (and all -1) are ground states satisfying all clauses.
         seed: Passed to :func:`numpy.random.default_rng()`, which is used
             to generate the clauses and the variable negations.
-
     Returns:
         A binary quadratic model with spin variables.
 
     .. note:: The clauses are randomly sampled from the space of 4-variable
         clauses *with replacement* which can result in collisions. However,
         collisions are allowed in standard problem definitions, are absent with
         high probability in interesting cases, and are almost always harmless
-        when they do occur.
+        when they do occur. Planting of a not all 1 ground state
+        can be achieved with a spin-reversal transform without loss of 
+        generality, planting can significantly modify the hardness of 
+        optimization problems. However, for large problems planting of a 
+        solution can be achieved (in the SAT phase) without modification of the
+        hardness qualities of the instance class. [#]_ 
+    .. [#] Lenka Zdeborova and Florent Krzakala,
```

Comment:
The placement of this citation (and others) looks a bit strange in the built docs 
![image](https://user-images.githubusercontent.com/8395238/170838154-231f9915-bc1f-4bc4-98d4-5243332079d1.png)

would recommend moving the other citation down to the bottom with it.

Also nice to have a space between the previous note and the citation for visual clarity